rosbag2/src/rosbag2/converter.cpp

```diff
@@ -72,4 +73,14 @@ std::shared_ptr<SerializedBagMessage> Converter::convert(
   return output_message;
 }
 
+void Converter::add_topic(const std::string & topic, const std::string & type)
+{
+  ConverterTypeSupport type_support;
+  type_support.type_support = get_typesupport(type, "rosidl_typesupport_cpp");
```

Comment:
I think I'd prefer having this renamed into something like `rmw_type_support` or something.